tests/ProcessManager/ChromeManagerTest.php

```diff
@@ -68,4 +68,22 @@ public function testMultipleInstances()
         $driver1->quit();
         $driver2->quit();
     }
+
+    public function testExtensionsAndExperimentalOptions()
+    {
+        $manager = new ChromeManager();
+        $manager->setExtensions([
+            __DIR__.'/../fixtures/chrome/extension1.crx',
+            __DIR__.'/../fixtures/chrome/extension2.crx',
```

Comment:
Are two extensions needed for this test? Would just a single extension work just as well?